safe/impact_functions/base.py

```diff
@@ -652,3 +692,197 @@ def message_pre_run(self):
             # KeyError is for when ['postprocessors'] is unavailable
             pass
         send_static_message(self, message)
+
+    def get_clip_parameters(self):
```

Comment:
``get_clip_parameters`` -> ``clip_parameters`` (generally I think it is better to avoid using ``get_`` prefixes for methods as it is verbose and usually does not add meaning).